serial/serial_resources/component_instance.py

```diff
@@ -0,0 +1,112 @@
+import serial
+from ..api_client import APIClient
+from .component_instance_link import ComponentInstanceLink
+
+class ComponentInstance:
+    def __init__(self, component_instance_data):
+        self.client = APIClient(serial.api_key, serial.base_url)
+        self.data = component_instance_data
+        self.links = [] 
```

Comment:
Thoughts on this being empty? I think maybe on get we should have a get links option so that this fills in. I get worried about storing an "incomplete" set of links here.

Reply:
I agree. I think we should only store this if it's accurate. If we don't explicitly get all links I think we should probably to set this `None` or not store it at all

Reply:
Decided to just rename in to created_links. I think we leave it like that for now